source/includes/stackpath/_workloads.md

```diff
@@ -117,6 +117,103 @@ Attributes | &nbsp;
 `isRemoteManagementEnabled`<br/>*boolean* | Specify if remote management is enabled on workload instance or not.
 `image`<br/>*string* | The workload's instance operating system image.
 
+<!-------------------- EDIT A WORKLOAD -------------------->
+
+### EDIT a workload
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+  -d "request_body" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads/1b932678-1038-4ab4-9fa4-c4c06e696e20"
+```
+> Request body example for a VM:
```

Comment:
Small suggestion to stay consistent with our [API docs template](https://raw.githubusercontent.com/cloudops/cloudmc-api-docs/cmc-dev/.github/PULL_REQUEST_TEMPLATE.md), wrap everything in a single code block of type `js` and use comments to distinguish between VMs and containers (we format as `js` instead of `json` else the comments appear in red)

```
> Request body example:
```js
// for a VM
{
   "name": "my-vm-workload",
   ...
}

// for a container
{
   "name": "my-container-workload",
   ...
}
```

Reply:
e4c7e8a, 6572183